### ISSUE TITLE
flas?

### ISSUE THREAD

**Issue by Isaagomen** [DuskieWhy/exe-2.5-code#8]
### Describe your problem here.

hello, i would like to know where are the flas  located, thanks.

### Are you modding a build from source or with Lua?

Source

### What is your build target?

Windows

**Comment by MemeHoovy** [comment]
pretty sure they never gave out FLAS

**Comment by MemeHoovy** [comment]
and I don't think they have plans of doing so anytime soon

**Comment by Isaagomen** [comment]
> and I don't think they have plans of doing so anytime soon

oh ok, because they published the 2.0 ones

**Comment by MemeHoovy** [comment]
yeah, but anything more then that, I don't think that's happening

**Comment by duckiewhy** [comment]
![bg](https://user-images.githubusercontent.com/88288894/199641370-67937e94-178e-4d82-bfa6-20d32fc4a77f.png)

**Comment by MemeHoovy** [comment]
penis
